schemas/iso19139/src/main/plugin/iso19139/process/thumbnail-remove.xsl

```diff
@@ -54,7 +54,6 @@
     <xsl:variable name="position" select="count(//gmd:graphicOverview[current() >> .]) + 1" />
 
     <xsl:if test="not(
-                      gmd:MD_BrowseGraphic[gmd:fileName/gco:CharacterString != ''] and
```

Comment:
Maybe try 
`gmd:MD_BrowseGraphic[normalize-space(gmd:fileName/gco:CharacterString) = normalize-space($thumbnail_url)] and`